client/src/components/Form/Elements/FormText.vue

```diff
@@ -118,6 +125,45 @@ export default {
                 : null;
         },
     },
+    watch: {
+        value(newValue) {
+            // Only update localValue from prop when not focused
+            if (!this.isFocused) {
+                this.localValue = this.valueToString(newValue);
+            }
+        },
+    },
+    mounted() {
+        // Initialize localValue with the initial prop value
+        this.localValue = this.valueToString(this.value);
+    },
+    methods: {
+        /**
+         * Converts the `FormText` value to a string representation.
+         * Handles arrays for multiple inputs and single values.
+         */
+        valueToString(v) {
+            const val = v ?? "";
+            if (Array.isArray(val)) {
+                if (val.length === 0) {
+                    return "";
+                }
+                return this.multiple
+                    ? val.reduce((str_value, item) => str_value + String(item) + "\n", "")
```

Comment:
this is (and was) broken btw, it'll add newlines as you type.
```suggestion
                    ? val.join("\n"): String(val[0]);
```